src/rules/media-feature-no-missing-punctuation/__tests__/index.js

```diff
@@ -61,6 +61,9 @@ testRule(rule, {
   }, {
     code: "@media screen and (max-width:($var - 42px)) {}",
     description: "ignore non-standard complex SCSS values",
+  }, {
+    code: "@media (max-width: $content-size + $side * 2) {}",
+    description: "ignore non-standard complex SCSS values",
   } ],
 
```

Comment:
This changes means I can do `@media (nothingreal: $content-size + $side * 2)`, right?

We should ensure that it *does not* allow this: `@media (nothingreal: $content-size)` — where no operator is involved.